testbook/execute.py

```diff
@@ -0,0 +1,20 @@
+from contextlib import contextmanager
+
+import nbformat
+import pytest
+
+from testbook.client import TestbookNotebookClient
+
+
+@pytest.fixture
+def notebook():
+    @contextmanager
+    def notebook_loader(nb_path, **kwargs):
+        with open(nb_path) as f:
+            nb = nbformat.read(f, as_version=4)
+
+        client = TestbookNotebookClient(nb)
+        with client.setup_kernel(**kwargs):
```

Comment:
I've got a PR that will make it so we can control the context and cleanup on user behalf without them needing use `with ..` syntax in their test. For now this is fund but we'll want to come back to it as unless the fixture handles cleanup for us then we might as well just let make this call `with TestbookNotebookClient(nb_path).setup(**kwargs) as nb:` work.